openedx/core/djangoapps/content_libraries/rest_api/containers.py

```diff
@@ -184,7 +184,8 @@ def get(self, request, container_key: LibraryContainerLocator):
             request.user,
             permissions.CAN_VIEW_THIS_CONTENT_LIBRARY,
         )
-        child_entities = api.get_container_children(container_key, published)
+        child_entities = api.get_container_children(container_key, published=published)
+        # TODO -- this looks backwards?
```

Comment:
I didn't get what you meant here. :sweat_smile:

Reply:
Ah sorry, I didn't mean to commit that comment :) I was confused when I wrote that, but it looks fine on second look.